src/current/_includes/releases/v23.1/v23.1.5.md

```diff
@@ -0,0 +1,93 @@
+## v23.1.5
+
+Release Date: July 5, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-5-security-updates">Security updates</h3>
+
+- The full set of TLS ciphers that was present in v22.1 have ben included in the old cipher suites list, which can be enabled with the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable. [#105370][#105370]
+
+<h3 id="v23-1-5-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Added new [cluster settings](../v23.1/cluster-settings.html) in the `server.oidc_authentication.generate_cluster_sso_token` namespace that support using OIDC to generate a [JWT auth token for cluster SSO](../v23.1/sso-sql.html). [#105643][#105643]
+
+<h3 id="v23-1-5-sql-language-changes">SQL language changes</h3>
+
+- Improved the inline documentation and error messages related to [`SHOW RANGES`](../v23.1/show-ranges.html). [#105278][#105278]
+- Allow inbound foreign keys on TTL tables. [#105341][#105341]
+- Added columns `default_value` and `origin` (default, override, external-override) to the [`SHOW CLUSTER SETTING`](../v23.1/show-cluster-setting.html) command. [#105452][#105452]
+- Added the `VIEWACTIVITY` and `VIEWACTIVITYREDACTED` [system privileges](../v23.1/security-reference/authorization.html#supported-privileges) for the `crdb_internal.cluster_queries` and `crdb_internal.node_queries` [tables](../v23.1/crdb-internal.html). [#105467][#105467]
+- Fixed the error message to be more understandable when attempting to create [UDFs](../v23.1/user-defined-functions.html) under a virtual or temporary schema. [#105627][#105627]
+
+<h3 id="v23-1-5-command-line-changes">Command-line changes</h3>
+
+- Running [`cockroch node decommission <nodeID>`](../v23.1/cockroach-node.html#node-decommission) for a node that has already been decommissioned will now exit with code 0, as had been the case in CockroachDB versions prior to v23.1.0. [#103933][#103933]
+
+<h3 id="v23-1-5-db-console-changes">DB Console changes</h3>
+
+- The [**Range Report** page](../v23.1/ui-hot-ranges-page.html#range-report) (route `/reports/range/:rangeID`) shows the **Hot Ranges** menu item as highlighted in the left side menu. The back button in the Range Report page redirects back to the Hot Ranges page. [#104980][#104980]
```

Comment:
Total nit:

```suggestion
- The [**Range Report** page](../v23.1/ui-hot-ranges-page.html#range-report) (route `/reports/range/:rangeID`) shows the **Hot Ranges** menu item as highlighted in the left-side menu. The back button in the Range Report page redirects back to the Hot Ranges page. [#104980][#104980]
```

Reply:
Also, I'm not 100% sure on whether the page names should be bold or not?

Reply:
Same for the "views" in the note below, should they be bold? Or, is that just GUI elements?